models/articles.models.js

```diff
@@ -1,5 +1,8 @@
 const db = require("../db/connection");
 
-exports.selectArticles = (id)=>{
-    return db.query(`SELECT * FROM articles WHERE article_id = $1`,[id])
-}
\ No newline at end of file
+exports.selectArticles = (id) => {
+  if (id < 1 || id > 13) {
+    return Promise.reject({ status: 404, msg: "article does not exist" });
+  }
```

Comment:
This isn't very dynamic. What if you wanted to post an article and an id of 14 was created? How could you check the database to see if the article id exists instead?